src/main/resources/portal.properties.EXAMPLE

```diff
@@ -202,3 +202,13 @@ ucsc.build=hg19
 
 # default view in oncoprint (sample, patient (default))
 oncoprint.defaultview=patient
+
+# labels to be displayed in oncoprint "Mutation color" menu for custom annotation of driver and passenger mutations in the oncoprint.
+# Set these properties to enable the respective menu options in oncoprint:
+# oncoprint.custom_driver_annotation.binary.menu_label=Custom binary annotation
+# oncoprint.custom_driver_annotation.class.menu_label=Custom tiers
+
+# set this to false to disable the automatic selection of the custom driver and passenger mutations annotation in the oncoprint 
+# (instead of OncoKB and Hotspots). By default this is true. Setting this to false will select OncoKB and Hotspots as the default 
+# annotation source instead.
+# oncoprint.custom_driver_annotation.default=false
```

Comment:
I first loaded test data without the properties. I think it's not correct to place the icons in this case, because I did not configure the customization correctly:

![screen shot 2017-08-14 at 11 27 27](https://user-images.githubusercontent.com/9624990/29266176-10f68764-80e4-11e7-8632-a66e90f61084.png)